lib/web_ui/lib/src/engine/skwasm/skwasm_impl/image.dart

```diff
@@ -9,8 +9,10 @@ import 'package:ui/src/engine.dart';
 import 'package:ui/src/engine/skwasm/skwasm_impl.dart';
 import 'package:ui/ui.dart' as ui;
 
-class SkwasmImage implements ui.Image {
-  SkwasmImage(this.handle);
+class SkwasmImage implements SkwasmObjectWrapper<RawImage>, ui.Image {
+  SkwasmImage(this.handle) {
```

Comment:
According to the `clone()` method below, it is the responsibility of the caller of the constructor to "pre-bump" the ref count of the handle. Does that hold for all invocations of this constructor?

Reply:
Basically yes. This constructor (and all skwasm object wrappers constructors that just take a raw handle) expect a handle that has +1 ref count on it, and the object basically takes ownership of that ref count. All the native `create` functions return a handle with +1 ref count on it, and when cloning we bump the ref count for this same reason.